cla_backend/apps/cla_butler/tests/test_tasks.py

```diff
@@ -127,7 +130,41 @@ def test_delete_old_data_run_case_over_two_years_successful_delete(self):
         case_complaints = Complaint.objects.filter(eod_id__in=eods).values_list("pk", flat=True)
         self.assertEqual(len(AuditLog.objects.filter(complaint__in=case_complaints)), 0)
 
-    def test_delete_old_data_run_case_under_two_years_unsuccessful_delete(self):
+    def test_delete_old_data_run_case_over_3_years_excluded_code_unsuccessful_delete(self):
+        """
+        This makes sure that even if a case is over 3 years
+        its not deleted due to having an excluded outcome_code
+        """
+        log = make_recipe("cla_auditlog.audit_log")
+
+        # Creating a case thats three years old so it gets picked up properly by the delete data
+        freezer = freeze_time(timezone.now() + relativedelta(years=-4))
+        freezer.start()
+        case = make_recipe("legalaid.case", audit_log=[log], outcome_code="SPOP")
+        freezer.stop()
+
+        eod = make_recipe("legalaid.eod_details", case=case)
```

Comment:
The assertion code for all three test methods is the same, can we refactor it into a common method where we pass in the expected number of object before/after to avoid duplication?